crates/holochain/src/core/ribosome/host_fn/call.rs

```diff
@@ -222,6 +224,12 @@ async fn execute_call(
                             ribosome_error.to_string()
                         ))
                         .into()),
+                        Err(ConductorApiError::ConductorError(ConductorError::CellMissing(
+                            cell_id,
+                        ))) => Err(wasm_error!(WasmErrorInner::Host(format!(
+                            "Call to missing cell: Cell with ID {cell_id} not found"
```

Comment:
The new specific error message when a bridge call references an unknown cell ID.